src/LineChart.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Chart;
+
+class LineChart extends Chart
```

Comment:
This complicates any extension in user projects... Not sure if different chart type that does not need additional modifications needs a custom class.

Reply:
There can be modifications in future is someone will be ready to contribute them :)

Reply:
I am fine with this for now.